LCM/scripts/OMS_MetaConfigHelper.py

```diff
@@ -4,6 +4,7 @@
 
 conf_path = "/etc/opt/microsoft/omsagent/conf/omsadmin.conf"
 metamof_path = "/etc/opt/omi/conf/omsconfig/generated_meta_config.mof"
+oms_agentid_path = "/etc/opt/microsoft/omsagent/agentid"
```

Comment:
when do we write agent ID at this path ? Is it when user onboard to first workspace ?